### PR TITLE
[MaxText] Change Gemma test path

### PR DESCRIPTION
**Pull request by khatwanimohit**
# Description

Change Gemma test path

# Tests

Ran locally: screen/A8M6sUEut8rzwJt

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [X] I have performed a self-review of my code.
- [X] I have necessary comments in my code, particularly in hard-to-understand areas.
- [X] I have run one-shot tests and provided workload links above if applicable. 
- [X] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: dags/multipod/maxtext_end_to_end.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
           tpu_cores=8,
           tpu_zone=Zone.US_CENTRAL2_B.value,
           time_out_in_min=60,
-          test_name=f"{test_name_prefix}-stable-{test_script}",
```
Comment (khatwanimohit) [inline dags/multipod/maxtext_end_to_end.py:51]: @jonb377 wdyt of this change ? I think in future the values of the dict are going to be paths so having keys in the test_name should be cleaner

File: dags/multipod/maxtext_end_to_end.py [hunk 2]
```diff
@@ -37,7 +37,7 @@
   test_models = {
       "llama2": ["test_llama2_7b"],
       "mistral": ["test_mistral"],
-      "gemma": ["test_gemma"],
```
[the remark this answers is not in the document]
Comment (RissyRan) [reply dags/multipod/maxtext_end_to_end.py:40]: Do you mean in the future if we have llama2-70b, we will name it as model llama2-70b-ckpt, llama2-70b-training, and llama2-70b-tuning. I think, we will have separate test scripts for a single large model.

**Review by RissyRan — COMMENTED** [review]
It will be good to have a quick test, either local or in composer env to ensure the changes do not break your tests. So, we have that checklist in the PR description to remind folks :)

File: dags/multipod/maxtext_end_to_end.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (khatwanimohit) [reply dags/multipod/maxtext_end_to_end.py:40]: Llama2-70b and Gemma-7B are the only models right now which will have two scripts ( 1 for cpu and other for tpu ). I think for that Anisha will have to create a seperate gke config function. So it understands which script to run on which cluster. 
Finetuning and training will always go in one script ( the tpu one ). 
In that case I'm imagining that key will look like `"llama2-70b": ["1_test.sh","2_test.sh"]`. In this scenario we'll not need the for loop and can pass the whole list in the gke config function.

[the remark this answers is not in the document]
Comment (RissyRan) [reply dags/multipod/maxtext_end_to_end.py:40]: This the format `"llama2-70b": ["1_test.sh","2_test.sh"]` will be align with my expectation. What do you mean using gke config function? 

I think for each script ("1_test.sh","2_test.sh"), we will need to call xpk to trigger the run. One for CPU and one for TPU? no?

**Comment by RissyRan** [comment]
Thanks for the test. Hmm..however, I cannot open it (with `https://screen/A8M6sUEut8rzwJt`).

**Review by RissyRan — APPROVED** [review]
Overall LGTM! Please ensure it's tested. Thanks!

File: dags/multipod/maxtext_end_to_end.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (khatwanimohit) [reply dags/multipod/maxtext_end_to_end.py:40]: I think there are multiple ways to do this, I'll let Anisha decide the best one. I was thinking we could hide to complexity of running multiple scripts for a model inside a new function created under `gke_config`

**Pull request merged**